File: src/main/java/com/livelazily/spring/learning/HelloController.java
package com.livelazily.spring.learning;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * Created by livelazily
 */
@RestController
public class HelloController {
    @RequestMapping("/")
    public String index() {
        return "Greetings from Spring Boot!";
    }
}

File: src/main/kotlin/com/livelazily/spring/learning/HelloController.kt
package com.livelazily.spring.learning

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 * Created by livelazily
 */
@RestController
class HelloController {
    @RequestMapping("/")
    fun index(): String {
        return "Greetings from Spring Boot!"
    }
}
